### DISCUSSION TITLE
load imagen from url .png or svg

### DISCUSSION THREAD

**Discussion by theerudito** [QuestPDF/QuestPDF#736] [General]
i need implement an image from url for example .png or svg please give me a solution

![image](https://github.com/QuestPDF/QuestPDF/assets/81595290/b75569d2-c972-407c-9840-d47b0ff067f7)

**Comment by girlpunk** [comment]
1. Download the image to a byte array
2. `row.Image(yourArray)`

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
